pages/api/getAudio.ts

```diff
@@ -0,0 +1,27 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { groq } from 'next-sanity';
+import { sanityClient } from '@/sanity';
+import { Audio } from '@/typings';
+
+const query = groq`
+    *[_type == "audio"]
+`;
+type Data = {
+  audio: Audio[];
+};
+
+export const getAudioStaticProps = async () => {
+  return await sanityClient.fetch<Audio[]>(query);
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data | ApiError>
+) {
+  try {
+    const audio = await getAudioStaticProps();
+    res.status(200).json({ audio });
+  } catch (err) {
+    res.status(500).json({ error: 'Error loading audio.' });
+  }
```

Comment:
All apis should be protected with a try catch block, this prevents a crash if there is an error from the server. 

if error is hit send the client a 500 response with a custom message.

Reply:
That helps tracking issues easier